src/test/polymer/polymer-element_test.ts

```diff
@@ -17,22 +17,33 @@ import {assert} from 'chai';
 import * as path from 'path';
 
 import {Analyzer} from '../../analyzer';
+import {HtmlScriptScanner} from '../../html/html-script-scanner';
 import {ClassScanner} from '../../javascript/class-scanner';
-import {Document} from '../../model/model';
+import {Document, Severity, Warning} from '../../model/model';
 import {PolymerElement} from '../../polymer/polymer-element';
+import {PolymerElementScanner} from '../../polymer/polymer-element-scanner';
 import {FSUrlLoader} from '../../url-loader/fs-url-loader';
 
 suite('PolymerElement', () => {
   const testFilesDir = path.resolve(__dirname, '../static/polymer2/');
   const urlLoader = new FSUrlLoader(testFilesDir);
   const analyzer = new Analyzer({
     urlLoader: urlLoader,
-    scanners: new Map([[
-      'js',
+    scanners: new Map<string, any[]>([
```

Comment:
@rictic for this test, there was a single scanner defined and I tried to add a minimum to get test to pass.  Would you recommend I just strip out the scanners option in this suite altogether?